agent/internal/client/methods.go

```diff
@@ -579,6 +579,23 @@ func (c *communicatorImpl) SendTestResults(ctx context.Context, taskData TaskDat
 	return nil
 }
 
+// SetHasResults sets the HasResults flag to true in the given task in the
+// database.
+func (c *communicatorImpl) SetHasResults(ctx context.Context, taskData TaskData) error {
```

Comment:
Rename this to match the variable name.

Reply:
What do you mean?

Reply:
`SetHasCedarResults` or `SetHasCedarTestResults`